### PR TITLE
@zephraph => Add <CurrentEvent /> and <Image /> primitive 

### PR DESCRIPTION
**Pull request by damassi**
current-event

### CODE REVIEW

File: src/Styleguide/Elements/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  SpaceProps,
+  WidthProps,
+  height,
+  HeightProps,
+} from "styled-system"
+
+export interface ImageProps extends SpaceProps, WidthProps, HeightProps {
+  src: string
+}
+export const Image = styled.img.attrs<ImageProps>({})`
```
Comment (zephraph) [inline src/Styleguide/Elements/Image.tsx]: 👏

File: src/Styleguide/Pages/Artist/Components/CurrentEvent.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  location: string
+  date: string
+}
+
+export class CurrentEvent extends React.Component<CurrentEventProps> {
+  render() {
+    return (
+      <Responsive>
+        {({ xs }) => {
+          if (xs) return null
```
Comment (zephraph) [inline src/Styleguide/Pages/Artist/Components/CurrentEvent.tsx]: Why does it not render on `xs`?

Comment (damassi) [reply src/Styleguide/Pages/Artist/Components/CurrentEvent.tsx]: Part of the spec is to hide it

File: src/Styleguide/Pages/Artist/__stories__/CurrentEvent.story.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { Section } from "../../../Utils/Section"
+import { CurrentEvent, LargeCurrentEvent } from "../Components/CurrentEvent"
+
+storiesOf("Styleguide/Artist", module).add("Current Event", () => {
+  return (
+    <React.Fragment>
+      <Section title="Responsive">
```
Comment (zephraph) [inline src/Styleguide/Pages/Artist/__stories__/CurrentEvent.story.tsx]: I'd started naming these just `Responsive` to begin with, but I think we should fully name them. `Responsive Current Event` or w/e. That way if you're taking screen shots or gifs or something it captures the full context.

Comment (damassi) [reply src/Styleguide/Pages/Artist/__stories__/CurrentEvent.story.tsx]: That makes sense, will update

**Comment by zephraph** [comment]
LGTM Merge on green

**Pull request merged**